Jenkinsfile.jssdk

```diff
@@ -40,8 +40,8 @@ ansiColor('xterm') {
           echo "Creating Branch"
           def version
           sh '''#!/bin/bash -ex
-          source ~/.nvm/nvm.sh > /dev/null 2>&1
-          nvm use v7
+          source ~/.nvm/nvm.sh
+          nvm use v8.9.1
```

Comment:
Do we want to use exact version or the `lts` tag?

Reply:
I'm just matching the version in the other jenkinsfile now. Haven't tested if `lts/*` works in jenkins yet. Will need a new PR to convert everything over.